packages/base/src/widget.ts

```diff
@@ -1101,14 +1101,18 @@ export class DOMWidgetView extends WidgetView {
   }
 
   updateTooltip(): void {
-    const title = this.model.get('tooltip');
+    const title = this.tooltip;
     if (!title) {
       this.el.removeAttribute('title');
-    } else if (this.model.get('description').length === 0) {
+    } else if (!this.model.get('description')) {
```

Comment:
currently the widget.tooltip is displayed only if the widget.description is empty. 
(tried various widgets such as text, checkbox, dropdown, on ipywidgets v8.1.5)

can we display the widget.tooltip even if the widget.description is not-empty?